environment.py

```diff
@@ -84,29 +84,42 @@ def findCellNeighbors(self):
                 self.grid[i][j].findNeighbors()
 
     def findCellsInCardinalRange(self, startX, startY, gridRange):
-        cellsInRange = []
+        cellsInRange = [{"cell": self.grid[startX][startY], "distance": 0}]
+        height = self.height
+        width = self.width
         for i in range(1, gridRange + 1):
-            deltaNorth = (startY + i + self.height) % self.height
-            deltaSouth = (startY - i + self.height) % self.height
-            deltaEast = (startX + i + self.width) % self.width
-            deltaWest = (startX - i + self.width) % self.width
-            cellsInRange.append({"cell": self.grid[startX][deltaNorth], "distance": i})
-            cellsInRange.append({"cell": self.grid[startX][deltaSouth], "distance": i})
-            cellsInRange.append({"cell": self.grid[deltaEast][startY], "distance": i})
-            cellsInRange.append({"cell": self.grid[deltaWest][startY], "distance": i})
+            deltaNorth = (startY + i + height) % height
+            deltaSouth = (startY - i + height) % height
+            deltaEast = (startX + i + width) % width
+            deltaWest = (startX - i + width) % width
+            cellsInRange.extend([
```

Comment:
If this can cleanly fit on 1-2 lines, please compress.

Reply:
Compressed in new commit, but I'm not sure if it's the style you're looking for. Let me know.